docs/tutorials/05_animation/viz_introduction.py

```diff
@@ -0,0 +1,92 @@
+"""
+===============================
+Keyframe animation introduction
+===============================
+
+This tutorial explains keyframe animation in FURY.
+
+"""
+
+###############################################################################
+# What is Keyframe Animation?
```

Comment:
This explanation still needs some simplification. Maybe something like this:

Animations in FURY

FURY provides an easy-to-use animation system that enables users creating complex animations based on keyframes. The user only need to provide the attributes of actors at certain times (keyframes), and the system will take care of animating everything through a interpolation.

[give an example]

you can continue your text with a title like "What exactly is a keyframe".